appUITests/Tests/AccountsTests/AccountsListTests.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  AccountsListTests.swift
+//  ios-golden-sample-app
+//
+//  Created by Backbase on 15/12/2023.
+//
+
+import Foundation
+import XCTest
+
+final class AccountsListTests : BaseTestCase {
+        
+    let testData = AccountsListTestData.self
+    
+    override func setUp() {
+        super.setUp()
+        LoginScreen(app).authenticateUserWith(name: UserEnrollment.userName, password: UserEnrollment.password)
+        WorkspacesSelectorScreen(app).selectServiceAgreementIfNeeded(workspaceName: UserEnrollment.boryCoffeeQA)
+    }
+    
+    func testAccountIsDisplayedInTheList() {
+        let accountListScreen = AccountsListScreen(app)
```

Comment:
should we use chaining 

```suggestion
       AccountsListScreen(app)
          .assertAccountScreenIsDisplayed()
          .accountListScreen.assertAccountIsDisplayed(name: testData.defaultAccountName, accountNumberEndsWith: testData.defaultAccountNumberEnds, balance: testData.defaultBalance)
```
in order to do this every function should either return new object of screen class or itself `Self`